ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryApiController.java

```diff
@@ -385,4 +391,47 @@ public String validate(User user, @RequestBody ScriptValidationParams scriptVali
 		fileEntry.setCreatedUser(user);
 		return scriptValidationService.validate(user, fileEntry, false, scriptValidationParams.getHostString());
 	}
+
+	/**
+	 * Download file entry of given path.
+	 *
+	 * @param user     current user
+	 * @param path     user
+	 * @param response response
+	 */
+	@GetMapping("/download/**")
+	public void download(User user, @RemainedPath String path, HttpServletResponse response) {
+		path = decodePathWithUTF8(path);
```

Comment:
Isn't it better to move this to RemainedPathArgumentHandler?

Reply:
I considered to decode it in the RemainedPathArgumentHandler but, I thought it was danger to decode all the remained path variable.

As I reconsider now, there seems to be no danger. Except the case that decodes remained path variable twice. I'm gonna move this and check the case.

Reply:
go for it

Reply:
You can make annotation have decode param. It will make it safe.
```
@RemainedPath(decodeWithUTF8=true)
```

Reply:
I inspected side effects when returning UTF-8 decoded RemainedPath variable. 

1. Upload File 
Get the file name from MultiPartFile object, so it doesn't need to be decoded. Get the current working directory as RemainedPath variable but, all directories are english. Also doesn't need to be decoded. 

2. Delete File 
Both v3.5.0 and before than v3.5.0, get file names to delete from request body(POST method). It doesn't need to be decoded. 

3. Download File 
Get target file from given RemainedPath variable. In this case, RemainedPath variable must be decoded. 

As a result, only when download a file must be decode RemainedPath variable. And there seems no side effect when returning UTF-8 decoded RemainedPath variable. 

I will move decoding step to RemainedPathResolver.

Reply:
By the way, use DELETE method for deleting file.

Reply:
Currently, it provides delete API in two methods. One for deleting a file, and the other for deleting multiple files.
https://github.com/naver/ngrinder/blob/48c08dacdc1d6d44c5710841c09787c3aa2e560e/ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryApiController.java#L256
https://github.com/naver/ngrinder/blob/48c08dacdc1d6d44c5710841c09787c3aa2e560e/ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryApiController.java#L370

When changed post to delete method, I think that `@DeleteMapping("/delete")` is redundant. 

How about deleting the first POST method API that used in front-end?

And make the `@DeleteMapping("/**")` API to work with both single file name and file name list. And change to use the delete method in front-end.

Reply:
https://github.com/naver/ngrinder/blob/48c08dacdc1d6d44c5710841c09787c3aa2e560e/ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryApiController.java#L370
I didn't recognize this API works for the given RemainedPath, I'm gonna change just POST API to use DELETE method